### PR TITLE
feat(pattern/radio): add radio input component and documentation

### PR DESCRIPTION
**Pull request by melanieanglardLMFR**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: MZC-54

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/pages/Components/Radio/previews/radio-disabled-state.preview.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ul class="example">
+    <li class="example__list-item">
+        <input id="radio-06" disabled="true" type="radio" class="mc-radio is-focus" name="example" />
```
Comment (gael-boyenval) [inline src/pages/Components/Radio/previews/radio-disabled-state.preview.html]: is-focus is wanted here ?

File: src/pages/Components/Radio/specs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| **Default** | min-height        | `20px`               |
+| **Default** | height            | `$mu125`             |
+
+### CSS Tokens : state colors
+
+> **You must always use the tokens, the colors are only there for that information.**
+
+| State                   | Property     | Token                              | Color   |
+| ----------------------- | ------------ | ---------------------------------- | ------- |
+| **Default**             | background   | `color.input.background`           | #ffffff |
```
Comment (gael-boyenval) [inline src/pages/Components/Radio/specs.md]: should we add the base token `color.primary...` or something like `color.primary... (#ffffff)` ?

File: src/pages/Components/Radio/previews/radio-disabled-state.preview.html (hunk 1 above)
Comment (melanieanglardLMFR) [reply src/pages/Components/Radio/previews/radio-disabled-state.preview.html]: @gael-boyenval no it's not necessary here

File: src/pages/Components/Radio/specs.md (hunk 2 above)
Comment (melanieanglardLMFR) [reply src/pages/Components/Radio/specs.md]: Not sure to understand you

**Pull request closed without merging**